### ISSUE TITLE
Repair USB-Deaddrop

### ISSUE THREAD

**Issue by uwekamper** [c-base/meta#46]
Was ist das? Siehe: http://deaddrops.com/

Im Durchgang mit Eingang zum ETI-Foyer war ein USB-Stick eingegipst. Nun ist er weg. Würde gernen nen neuen hin machen.

**Comment by XenGi** [comment]
Besteht da noch Interesse?

**Comment by uwekamper** [comment]
Ja, ich will das Projekt immer noch haben. Aber wir müssen es an einer versteckteren Stelle einbauen. Vor dem ETI (da wo es zuletzt war) hängen zu viele Leute rum. Dort wird es abgebrochen.

**Comment by XenGi** [comment]
Im aboretum im Sockel war auch mal einer. Wir könnten natürlich einen in der c-base machen aber der Sinn von deaddrops ist ja eigentlich das sie zu jederzeit erreichbar sind.
Ich hätte da auch immer noch Bock drauf also wenn sich eine Stelle findet wär ich dabei.

**Comment by berlincount** [comment]
Oder man baut ihn tiefer ein, so dass er nur mit http://i.ebayimg.com/00/s/MTAwMFgxMDAw/z/d5wAAOSwsB9V-ubM/$_35.JPG?set_id=8800005007 benutzbar wird ;)

**Comment by ri0t** [comment]
Hey freunde - schonmal von der Technologie "Verlängerungskabel" gehört? Ja, das limitiert etwas den use-case - weil man eins dabei haben muss. Who cares?

**Comment by bergie** [comment]
We've got a USB deaddrop to IPFS bridge running in Siri:

https://github.com/c-base/ipfs-deaddrop

However, the old version of IPFS we're running on the system is somewhat problematic, and causes the computer to hang. We should upgrade
